doc/cephfs/cephfs-mirroring.rst

```diff
@@ -93,6 +93,15 @@ providing high-availability.
 .. note:: Deploying a single mirror daemon is recommended. Running multiple
    daemons is untested.
 
+The following file types are supported by the mirroring:
+
+- Regular files (-)
+- Directory files (d)
+- Symbolic link file (l)
+
+The other file types are ignored by the mirroring. So they won't be
```

Comment:
If it isn't mentioned already, please add a note about hardlinked files too.

Reply:
Oops, will do that.